src/components/appRouter.js

```diff
@@ -252,20 +195,14 @@ class AppRouter {
         setBackdropTransparency(level);
     }
 
-    handleConnectionResult(result) {
+    #handleConnectionResult(result) {
         switch (result.State) {
             case 'SignedIn':
                 loading.hide();
                 this.goHome();
                 break;
             case 'ServerSignIn':
-                result.ApiClient.getPublicUsers().then((users) => {
```

Comment:
This API call is not needed since we don't even use the second parameter in the `showLocalLogin` method and we already have the current server ID.